decentraland/unity-renderer
# feat: remove white texture from entity information and fixed undo/red…

## What this PR includes?


-  EntityListDetails have a white texture in the texture of the item

-  If you made changes and exit from builder in world and enter again, the undo/redo button is not updated (visual effect that tell you that you are able to undo/redo)

## How to test it?

1. Go to: https://play.decentraland.zone/?index.html&renderer=urn:decentraland:off-chain:renderer-artifacts:fix/biw-bug-fixing-2&ENABLE_BUILDER_IN_WORLD
2. Enter in builder in world 
3. Select and entity and check the white texture
4. Made a change in the scene and exit
5. Enter again and check the the undo/redo

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
